frontend/src/metabase/admin/upsells/components/UpsellCard.tsx

```diff
@@ -81,15 +96,24 @@ export const _UpsellCard: React.FC<UpsellCardProps> = ({
           <Text lh="1rem" px="1rem">
             {children}
           </Text>
-          <Box
-            component={ExternalLink}
-            onClickCapture={() => trackUpsellClicked({ source, campaign })}
-            href={url}
-            className={S.UpsellCTALink}
-            mx="md"
-            mb="lg"
-          >
-            {buttonText}
+          <Box mx="md" mb="lg">
+            {buttonLink ? (
+              <ExternalLink
+                onClickCapture={() => trackUpsellClicked({ source, campaign })}
+                href={url}
+                className={S.UpsellCTALink}
+              >
+                {buttonText}
+              </ExternalLink>
+            ) : (
+              <Link
+                onClickCapture={() => trackUpsellClicked({ source, campaign })}
+                to={internalLink as string}
```

Comment:
nit: I think you can avoid this `as` if you change line 100 to 

```ts
            {buttonLink !== undefined ? (
```

Reply:
And you are 100% correct :)
Thank you
https://github.com/metabase/metabase/pull/53951/commits/fa4597f12436f9cc7f574df1d737039b4381d9e2